### PR TITLE
Cleanup IM command

### PR DESCRIPTION
**Pull request by yunhanw-google**
Summary of Changes:
--use enum class
--rename some parameter
--eliminate duplicate code
--cleanup comments

### CODE REVIEW

**Review by mspang — APPROVED** [review]

**Review by mspang — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Review by woody-apple — APPROVED** [review]

**Comment by woody-apple** [comment]
@saurabhst @jelderton ?

File: src/app/Command.h [hunk 1]
```diff
@@ -48,18 +45,18 @@ namespace app {
 class Command
 {
 public:
-    enum CommandRoleId
+    enum class CommandRoleId
     {
-        kCommandSenderId  = 0,
-        kCommandHandlerId = 1,
+        SenderId  = 0,
+        HandlerId = 1,
```
Comment (bzbarsky-apple) [inline src/app/Command.h:51]: Maybe "Sender" and "Handler"?  Otherwise you get `CommandRoleId::SenderId` which is a bit repetitive...

File: src/app/InteractionModelEngine.h [hunk 2]
```diff
@@ -50,6 +50,9 @@
 namespace chip {
 namespace app {
 
+constexpr size_t kMaxSecureSduLength  = 1024;
```
Comment (bzbarsky-apple) [inline src/app/InteractionModelEngine.h:53]: Please document that this is the size (of what?) in bytes.

File: src/app/InteractionModelEngine.h [hunk 3]
```diff
@@ -50,6 +50,9 @@
...
+constexpr uint32_t kImMesssageTimeout = 20;
```
Comment (bzbarsky-apple) [inline src/app/InteractionModelEngine.h:54]: Please document, especially units.  Also, why the change from 5 to 20?

**Review by bzbarsky-apple — APPROVED** [review]

**Review by saurabhst — APPROVED** [review]

File: src/app/InteractionModelEngine.h (hunk 3 above)
Comment (yunhanw-google) [reply src/app/InteractionModelEngine.h:54]: 20 is accidental change, would update it back to 5 in next pr

File: src/app/Command.h (hunk 1 above)
Comment (yunhanw-google) [reply src/app/Command.h:51]: sure, will do it in next cleanup pr

File: src/app/InteractionModelEngine.h (hunk 2 above)
Comment (yunhanw-google) [reply src/app/InteractionModelEngine.h:53]: sure，will do it in next pr
